fmf/utils.py

```diff
@@ -912,34 +915,33 @@ def dict_to_yaml(data, width=None, sort=False):
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 class JsonSchemaValidationResult(NamedTuple):
-    """ Represents JSON Schema validation result """
-
+    """Represents JSON Schema validation result."""
     result: bool
-    errors: List[Any]
+    errors: list[Any]
 
 
-def validate_data(data, schema, schema_store=None):
+def validate_data(
+        data: Any,
+        schema: Any,
+        schema_store: Optional[dict[str, Any]] = None
+        ) -> JsonSchemaValidationResult:
     """
     Validate data with given JSON Schema and schema references.
 
-    schema_store is a dict of schema references and their content.
-
-    Return a named tuple utils.JsonSchemaValidationResult
-    with the following two items:
+    Args:
+        data: The data to validate
+        schema: The JSON Schema to validate against
+        schema_store: Optional dict of schema references and their content
 
-        result ... boolean representing the validation result
-        errors ... A list of validation errors
+    Returns:
+        JsonSchemaValidationResult with:
+            result: boolean representing the validation result
+            errors: A list of validation errors
```

Comment:
What docstyle has been adopted?

Reply:
hah, you have to ask my IDE, I'm just pressing TAB :D  
Joking aside, I do rely on formatters and linters for this stuff. :/

Reply:
@LecrisUT maybe we can move https://github.com/teemtee/fmf/pull/240 forward?

Reply:
> @LecrisUT maybe we can move #240 forward?

Yeah, let's enable it in `1.6`.